presto-native-execution/presto_cpp/main/PrestoServer.h

```diff
@@ -264,6 +270,9 @@ class PrestoServer {
   folly::Synchronized<bool> shuttingDown_{false};
   std::chrono::steady_clock::time_point start_;
   std::unique_ptr<PeriodicTaskManager> periodicTaskManager_;
+#ifdef PRESTO_ENABLE_LINUX_MEMORY_CHECKER
+  std::unique_ptr<LinuxMemoryChecker> linuxMemoryChecker_;
```

Comment:
To avoid this, can we use a static inside `PrestoServer.cpp`?

Reply:
@majetideepak I'm not sure what you mean, can you maybe elaborate on this?

Reply:
You can create a global inside a CPP file. See
https://github.com/prestodb/presto/blob/1f74d4d96191775abde662e64a246ff1f2dd56bf/presto-native-execution/presto_cpp/presto_protocol/ConnectorProtocol.cpp#L21